app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt

```diff
@@ -81,14 +79,10 @@ class NotificationRegistrar @Inject constructor(
         appCoroutineScope.launch { registerApp() }
     }
 
-    @Suppress("NewApi") // we use appBuildConfig
+    @Suppress("NewApi") // we use NotificationCompatManager to retrieve channels
     override fun onResume(owner: LifecycleOwner) {
         val systemEnabled = compatManager.areNotificationsEnabled()
-        val allChannelsEnabled = when {
-            appBuildConfig.sdkInt >= O -> manager.notificationChannels.all { it.importance != IMPORTANCE_NONE }
-            else -> true
-        }
-
+        val allChannelsEnabled = compatManager.notificationChannels.all { it.importance != IMPORTANCE_NONE }
```

Comment:
Removed the `when` as now we're using the `NotificationCompatManager` and `allChannelsEnabled` returns an empty collection for OS versions < O. This means `allChannelsEnabled=true` for OS versions < O.